src/main/java/de/tum/in/www1/artemis/config/SAML2Configuration.java

```diff
@@ -55,7 +48,7 @@ public class SAML2Configuration extends WebSecurityConfigurerAdapter {
      */
     public SAML2Configuration(final SAML2Properties properties) {
         this.properties = properties;
-
+        // TODO: we should describe why this line is actually needed
```

Comment:
yes, please explain

Reply:
@dfuchss Could you assist here? It's been a TODO that Stephan added, but as you added the LoC below and are the SAML2 expert here, maybe you help me here. Feel free to just answer and I add the explanation to the codebase.

Reply:
We can solve this in a follow up